src/app/lists/ListDataview.tsx

```diff
@@ -123,12 +124,13 @@ const ListsDataView: React.FC<ListDataViewProps> = ({
     listOfIngredients,
     setListOfIngredients,
     comment,
+    error,
 }) => {
     const [modal, setModal] = useState<EditModalState>();
     const [toDelete, setToDelete] = useState<number | null>(null);
     // need another setState otherwise the modal content changes before the close animation finishes
     const [toDeleteModalOpen, setToDeleteModalOpen] = useState<boolean>(false);
-    const [errorMsg, setErrorMsg] = useState<string | null>(null);
+    const [errorMessage, setErrorMessage] = useState<string | null>(error);
```

Comment:
So it seems like we manage the state of error messages in ListsPage and ListsDataView? Is this intended?

We should let ListsPage manage the state and pass `errorMessage` and `setErrorMessage` down to `ListsDataView` maybe?